server/services/weather.py

```
@@ -0,0 +1,124 @@
+        resp = httpx.get(
+            "https://api.openweathermap.org/data/2.5/weather",
+            params={"q": city, "appid": api_key, "units": "metric"},
+            timeout=10,
+        )
+        resp.raise_for_status()
+        data = resp.json()
+
+        result = WeatherResult(
+            city=data["name"],
+            temperature_c=data["main"]["temp"],
+            timestamp=datetime.utcnow(),
```

Ruff (0.15.6)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)